spec/lib/seed_data/categories_spec.rb

```diff
@@ -135,15 +135,15 @@ def permissions(group, type)
     end
 
     it "does not override permissions of existing category when not forced" do
-      create_category("lounge_category_id")
+      create_category("general_category_id")
```

Comment:
The name is not relevant to what is tested here, but find it's better to not have references to deprecated stuff, even if they are soft. 🙂